### DISCUSSION TITLE
New Google Maps Colors

### DISCUSSION THREAD

**Discussion by henrikzabel** [react-native-maps/react-native-maps#5166] [Q&A] [answered]
I was wondering, why the current map in my app is not showing the new colors of Google Maps. When opening the Google Maps App on the same device the new colors are showing up but my app still looks like 2016:

![image](https://github.com/user-attachments/assets/632f37a0-976c-4086-a795-85b8531403c8)

**Comment by salah-ghanim** [comment, marked as answer]
@HenrikZabel This library should have 1:1 implementation to the Google Maps SDK not the Google Maps app, still a new version of the GoogleMaps SDK is now available with the latest react-native-maps, maybe that solves it, else try to use custom google maps using googleMapID to setup a similar look to the GoogleMaps app.
